### ISSUE TITLE
Bug with maintaining bank

### ISSUE THREAD

**Issue by badatnames** [greenc/CookieMaster#69]
I have been experiencing a bug with auto-buy and maintain bank enabled, where it will not always properly maintain my bank.

To be clear, I am absolutely _not_ referring to when a purchase raises the required bank, causing it to 'leapfrog' over my current bank.  I understand that that is working as intended, as maintain bank is not really, in the strictest sense, meant to really maintain the maximum bank, but rather to ensure that a purchase would never result in a lower reward from a golden cookie.

That said, the bug I am experiencing is literally the auto-buyer ignoring the fact that it should maintain bank at certain times.  I sat down to watch the progression for a while to see if I could figure out more specifically what would trigger this, and it actually seems to be a frenzy occurring.  The moment a frenzy begins, the auto-buy seems to buy the next thing in the queue, so long as it can afford it, disregarding whether that purchase would maintain bank or not.  At all other times, so far as I have been able to observe, maintain bank appears to function as intended.

**Comment by keegansimyh** [comment]
Yes, it is caused by a frenzy. When a frenzy occurs, sometimes the "Lucky+Frenzy Bank" drops to 1/7. Might be the game thinking when its a frenzy the base cps is 1/7 as much, when it actually isn't. Turned off auto-buy for now.
